specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -4205,6 +4203,25 @@
         "configurationPath": {
           "description": "The configuration file of the package execution. Type: string (or Expression with resultType string).",
           "type": "object"
+        },
+        "packageName": {
```

Comment:
adding new properties is suggested to be added in service tag.

Reply:
thank you for review. Could you give me an example about servie tag? It seems a new requriement.

Reply:
service tag is a new version like 2019-11-01

Reply:
I am not doing breaking change, it won't break exsiting user's scenario. Refer to merged PR like: [Add a alternateKeyName property in dynamics sink](https://github.com/Azure/azure-rest-api-specs/commit/e44be3308e43330571fef08d7f0bfdacc4801346#diff-c41ebf893eda93fe86884a699f9606c4) it does not change service tag too